--- xevd-0.4.1-4e76654c/src_main/CMakeLists.txt.orig	2022-12-21 06:54:35.000000000 +0100
+++ xevd-0.4.1-4e76654c/src_main/CMakeLists.txt	2024-04-13 19:47:21.643990768 +0200
@@ -115,6 +115,7 @@ elseif( UNIX OR MINGW )
                                                         LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
     target_compile_definitions( ${LIB_NAME} PUBLIC ANY LINUX )
     target_link_libraries(${LIB_NAME} m)
+    target_link_libraries(${LIB_NAME}_dynamic m)
 endif()
 
 # Install rules
--- xevd-0.4.1-4e76654c/src_base/CMakeLists.txt.orig	2022-12-21 06:54:35.000000000 +0100
+++ xevd-0.4.1-4e76654c/src_base/CMakeLists.txt	2024-04-13 19:47:49.867171203 +0200
@@ -94,6 +94,7 @@ elseif( UNIX OR MINGW )
                                                             LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
     target_compile_definitions( ${LIB_NAME_BASE} PUBLIC ANY LINUX )
     target_link_libraries(${LIB_NAME_BASE} m)
+    target_link_libraries(${LIB_NAME_BASE}_dynamic m)
 endif()
 
 # Install rules
