--- xevd-0.4.1-4e76654c/CMakeLists.txt.orig	2024-04-13 17:49:01.992452880 +0200
+++ xevd-0.4.1-4e76654c/CMakeLists.txt	2024-04-13 17:55:36.726981086 +0200
@@ -106,7 +106,7 @@ elseif( UNIX OR MINGW )
     endif()
 
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -${OPT_LV} ${OPT_DBG} -fomit-frame-pointer -Wall -Wno-unused-function -Wno-unused-but-set-variable -Wno-unused-variable -Wno-attributes -Werror -Wno-unknown-pragmas -Wno-stringop-overflow -std=c99")
-    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-pointer-sign -pthread -Wno-pointer-to-int-cast -Wno-maybe-uninitialized")
+    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-pointer-sign -pthread -Wno-pointer-to-int-cast -Wno-maybe-uninitialized -Wno-format-truncation")
     set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lm")
 endif()
 
--- xevd-0.4.1-4e76654c/src_base/xevd_port.c.orig	2022-12-21 06:54:35.000000000 +0100
+++ xevd-0.4.1-4e76654c/src_base/xevd_port.c	2024-04-13 17:47:12.533045873 +0200
@@ -49,7 +49,7 @@ void xevd_trace_line(char * pre)
     int len = (pre == NULL) ? 0 : (int)strlen(pre);
     if (len > 0)
     {
-        sprintf(str, "%s ", pre);
+        snprintf(str, 128, "%s ", pre);
         len = (int)strlen(str);
     }
     for (int i = len; i < chars; i++) { str[i] = '='; }
--- xevd-0.4.1-4e76654c/app/xevd_app_util.h.orig	2022-12-21 06:54:35.000000000 +0100
+++ xevd-0.4.1-4e76654c/app/xevd_app_util.h	2024-04-13 18:05:49.166996547 +0200
@@ -67,7 +67,7 @@ static void log_line(char * pre)
     len = (pre == NULL)? 0: (int)strlen(pre);
     if(len > 0)
     {
-        sprintf(str + 3, " %s ", pre);
+        snprintf(str + 3, 125, " %s ", pre);
         len = (int)strlen(str);
     }
 
